samples/Mvc.RenderViewToString/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using Microsoft.AspNet.FileProviders;
+using Microsoft.AspNet.Hosting;
+using Microsoft.AspNet.Hosting.Internal;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Mvc.RenderViewToString
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Initialize the necessary services
+            var services = new ServiceCollection();
+            ConfigureDefaultServices(services, path: null);
+
+            // Add a custom service that is used in the view.
+            services.AddSingleton<EmailReportHelper>();
+
+            var provider = services.BuildServiceProvider();
+            var helper = provider.GetRequiredService<RazorViewHelper>();
+
+            var model = new EmailViewModel
+            {
+                UserName = "User",
+                SenderName = "Sender",
+                UserData1 = 1,
+                UserData2 = 2
+            };
+            var emailContent = helper.RenderViewToString("EmailTemplate", model);
+            Console.WriteLine(emailContent);
+        }
+
+        private static void ConfigureDefaultServices(IServiceCollection services, string path)
```

Comment:
I think `path` here requires a little more fanfare for a reader to understand exactly what this does.